src/views/Home.vue

```diff
@@ -144,6 +154,12 @@ export default class Home extends Vue {
             ' <strong class="required">(' + requiredText + ")</strong>";
         }
 
+        if (options.question.help) {
+          let helpTxt = sender.locale == "fr" ? String(options.question.help.fr) : String(options.question.help.default);
+          let showHelp = sender.locale == "fr" ? "Afficher l'aide" : "Show help"; 
```

Comment:
Please externalize these strings to en.json and fr.json, using this.$t("key") to get the translation. This keeps all translations in one spot.

Reply:
Fixed.